### PR TITLE
[zh] Localize 'api_gateway'

### LINKED ISSUE #850: [zh] Localize 'api_gateway'
- [ ] Localize the content/en/api_gateway.md to content/zh/ in dev-zh.

### PR DESCRIPTION
**Pull request by Anddd7**
Localize the `content/en/api_gateway.md`, fix #850

### CODE REVIEW

File: content/zh-cn/api_gateway.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: API 网关
+status: Completed
+category: 技术
+---
+
+
+## 是什么
+
+[API](/application_programming_interface/) 网关是一种通过聚合多个应用程序的 API，并实现一站式管理的工具。它允许组织将关键性功能移交到一个可集中管理的地方，例如身份验证和授权、限制应用程序之间的请求数量。一个 API 网关则作为一个公共的接口，向API消费者（通常来自外部）提供服务。
```
Comment (hanyuancheung) [inline content/zh-cn/api_gateway.md:10]:
```suggestion
[API 网关](/application_programming_interface/)是一种通过聚合多个应用程序的 API，并实现一站式管理的工具。它允许组织将关键性功能移交到一个可集中管理的地方，例如身份验证和授权、限制应用程序之间的请求数量。一个 API 网关则作为一个公共的接口，向API消费者（通常来自外部）提供服务。
```

**Review by hanyuancheung — APPROVED** [review]
lgtm.

File: content/zh-cn/api_gateway.md (hunk 1 above)
Comment (Anddd7) [reply content/zh-cn/api_gateway.md]: modified, please take a look

**Review by Rocksnake — APPROVED** [review]
lgtm

**Review by Jacob953 — APPROVED** [review]
LGTM.

**Pull request merged**